diesel/src/migration/mod.rs

```diff
@@ -0,0 +1,55 @@
+//! Representation of migrations
+
+pub mod errors;
+pub use self::errors::*;
```

Comment:
This looks super dangerous adds a hard to see amount of items to our public API!

I does in fact only expose `MigrationError` and `RunMigrationsError`. So let's `pub use` those specifically if needed.